src/ol/mapbrowserevent.js

```diff
@@ -37,25 +36,19 @@ ol.MapBrowserEvent = function(type, map, browserEvent, opt_dragging,
   goog.base(this, type, map, opt_frameState);
 
   /**
-   * @const
-   * @type {goog.events.BrowserEvent}
-   */
-  this.browserEvent = browserEvent;
```

Comment:
Glad to see this gone.

For consistency, I think it would be nice if `ol.pointer.PointerEvent` also had an `originalEvent` property instead of `browserEvent`.  I know this goes beyond the changes you are making here, but if you agree, you can cherry-pick from my [`original-events` branch](https://github.com/ahocevar/ol3/compare/no-goog-events...tschaub:original-event) (this is mostly argument/variable renaming for the sake of readability).